### ISSUE TITLE
Increase performance of "property" coalescence

### ISSUE THREAD

**Issue by cbizon** [ranking-agent/AnswerCoalesce#12]
Given the following question:
```
"nodes": [
        {
            "id": "a",
            "type": "disease",
            "curie": "MONDO:0005015"
        },
        {
            "id": "b",
            "type": "gene"
        },
        {
            "id": "c",
            "type": "chemical_substance"
        }
    ],
    "edges": [
        {
            "id": "ab",
            "source_id": "a",
            "target_id": "b",
            "type": "gene_associated_with_condition",
        },
        {
            "id": "bc",
            "source_id": "c",
            "target_id": "b",
            "type": "decreases_activity_of"
        }
    ]
    }
```

If you run this through strider, and wait for it to complete, you get O(500) answers.  Sending this to property coalesence does not complete in 2 hours.   I suspect that this could be helped through batching calls to the sqlite db, but there may be other issues.

**Comment by PhillipsOwen** [comment]
the above JSON is invalid and the exception was not trapped. added JSONDecodeError to exception handler. fixed in eafd7c47857ba6ec99e2dc6bdd14d86357e7898f

**Comment by PhillipsOwen** [comment]
This query also fails in code. exception thrown in function identify_coalescent_nodes(answerset) because nodes     

question = answerset['query_graph']
graph = answerset['knowledge_graph']

are not in the initial JSON request.

**Comment by cbizon** [comment]
My issue was not specific enough.   The actual input should be a fully constructed message with the above snippet as the 'query_graph' value (minus the stray comma)

**Comment by cbizon** [comment]
I believe that this has been fixed / improved.  Please confirm and if true, then close.

**Comment by PhillipsOwen** [comment]
fixed in a4a494d

**Comment by PhillipsOwen** [comment]
new tests passed.
